cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerTwillRunnable.java

```diff
@@ -177,6 +191,10 @@ private void doInitialize(TwillContext context) throws Exception {
                                                               Constants.Logging.COMPONENT_NAME,
                                                               TaskWorkerTwillApplication.NAME);
     LoggingContextAccessor.setLoggingContext(loggingContext);
+
+    tokenManager = injector.getInstance(TokenManager.class);
```

Comment:
I feel a bit wrong for the Task worker needing to start the token manager since it requires the `security.data.keyfile.path` being mount into this container, which is supposed to be running user code. If that secret is mounted, it means the user code can call any service? @dli357

Reply:
It would depend on how we want to propagate identity. Right now, the taskworkers are still sending placeholder credentials when they make calls to any other service which seems kind of hacky; it might be a good idea to have them mount their own secret in the future. However, as it is right now, they do not need to start the token manager.

Reply:
Ok. Removing it for now.